README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Build Status 
 
-OV Version | 5.60 | 5.50 | 5.40 | 5.30 |
+OV Version | 6.0 | 5.60 | 5.50 | 5.40 |
 | ------------- |:-------------:| -------------:| -------------:| -------------:|
-SDK Version/Tag | [Master](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/master) | [v5.17.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.17.0) | [v5.16.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.16.0) | [v5.15.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.15.0) |
+SDK Version/Tag | [Master](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/master) | [v5.18.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.18.0) | [v5.17.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.17.0) | [v5.16.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.16.0) |
```

Comment:
change master to 6.0.0 and change the respective url. I kept it as master by mistake in previous release, please correct now.